src/EnergyPlus/WaterCoils.hh

```diff
@@ -273,6 +273,10 @@ namespace WaterCoils {
 		Real64 FoulingFactor; // Coil fouling factor [m2K/W]
 		bool DesiccantRegenerationCoil; // true if it is a regeneration air heating coil defined in Desiccant Dehumidifier system
 		int DesiccantDehumNum; // index to desiccant dehumidifier object
+		// variables for coil fault
+		bool FaultyCoilSATFlag; // True if the coil has SAT sensor fault
```

Comment:
@rongpengzhang I don't think these new variables are used, because the faults are applied in the water coil controller.